content/en/docs/setup/certificates.md

```diff
@@ -0,0 +1,136 @@
+---
+title: PKI Certificates and Requirements
+reviewers:
+- sig-cluster-lifecycle 
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+Kubernetes requires PKI certificates for authentication over TLS.
+If you install Kubernetes with [kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm/), the certificates that your cluster requires are automatically generated.
+You can also generate your own certificates -- for example, to keep your private keys more secure by not storing them on the API server.
+This page explains the certificates that your cluster requires.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## How certificates are used by your cluster
+
+Kubernetes requires PKI for the following operations:
+
+* Client certificates for the kubelet to authenticate to the API server
+* Server certificate for the API server endpoint
+* Client certificates for administrators of the cluster to authenticate to the API server
+* Client certificates for the API server to talk to the kubelets
+* Client certificate for the API server to talk to etcd
+* Client certificate/kubeconfig for the controller manager to talk to the API server
+* Client certificate/kubeconfig for the scheduler to talk to the API server.
+* Client and server certificates for the [front-proxy][proxy]
+
+etcd also implements mutual TLS to authenticate clients and peers.
+
+## Where certificates are stored
+
+Kubernetes stores certificates by default in `/etc/kubernetes/pki`. All paths in this documentation are relative to that directory.
```

Comment:
It might be good to clarify that this is the default for kubeadm-installed clusters specifically.

Reply:
done